tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -1271,8 +1271,20 @@ struct QueryPlanRequest {
 }
 
 struct QueryPlanResponse {
-  strategy @0 :Text;
```

Comment:
I think removing fields from a capnp struct is a breaking change, but maybe it doesn't matter here since query plans are not yet implemented in Cloud, right?

Reply:
This hasn't shipped yet. We are fine!